src/components/VariantDb/StatsTable.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { Col, List, Row } from 'antd';
+import Title from 'antd/es/typography/Title';
+import { fetchVariantDBStats } from './fetchVariantStats';
+
+type VariantDbStats = {
+  distinctVariantsCount: number;
+  occurrencesCount: number;
+  participantsCount: number;
+  studiesCount: number;
+};
+
+const initialVariantStatsState = [
+  {
+    name: 'Studies',
+    value: 0,
+  },
+  {
+    name: 'Participants',
+    value: <div>-</div>,
+  },
+  {
+    name: 'Distinct Variants',
+    value: 0,
+  },
+  {
+    name: 'Occurrences',
+    value: 0,
+  },
+];
+
+const StatsTable = (): JSX.Element => {
+  const [statsData, setStatsData] = useState(initialVariantStatsState);
+
+  useEffect(() => {
+    async function fetchStats() {
+      try {
+        const stats = await fetchVariantDBStats();
+        const sourceDoc: VariantDbStats = stats._source;
+
+        const data = [
+          {
+            name: 'Studies',
+            value: sourceDoc.studiesCount,
+          },
+          {
+            name: 'Participants',
+            value: <div>{sourceDoc.participantsCount}</div>,
```

Comment:
I m not a fan of putting dom elements in the state. But I don't have a justification for it. Maybe check with the others if my concern is well-founded or not?